atom/browser/ui/cocoa/atom_bundle_mover.mm

```diff
@@ -74,21 +74,17 @@
     if ([fileManager fileExistsAtPath:destinationPath]) {
       // But first, make sure that it's not running
       if (IsApplicationAtPathRunning(destinationPath)) {
-        // Give the running app focus and terminate myself
-        [[NSTask
```

Comment:
I'd like to retain the ability for folks to do this, I believe it is the sensible default behavior.  Making it overridable / interceptable would make sense but if we remove this apps can't do this without either string matching the error message (which you should never do) and then spawning `open` themselves.

Not sure what the ideal state looks like but retaining this activate-and-quit behavior is important I feel.  i.e. the 99% case is activate-and-quit, we shouldn't remove that to fit the 1% case

Reply:
🤔 maybe we can just pass an (optional) bool that determines whether to disable this behavior if there's a conflict?

Reply:
I'd love to give app devs the chance to let users decide at-conflict time rather than in advance.  E.g. an API that looked roughly like this.

```js
app.moveToApplicationsDirectory({
  conflictHandler: (conflictType) => {
    if (conflictType === 'exists') {
      return askUserIfShouldOpenAndQuit()
    }
    return true;
  }
})
```

Disclaimer: super rough sketch

Reply:
sgtm, will work on this at a Work Time next week 😆